app/models/ems_refresh/save_inventory.rb

```diff
@@ -115,7 +115,7 @@ def save_vms_inventory(ems, hashes, target = nil)
 
         found.save!
         h[:id] = found.id
-        h[:_object] = found
+        h[:_object] = found.reload
```

Comment:
It was added to do a similar thing that setting h[:id] does, but just keeps the whole object in the hash.  That way it can be referenced later by just using it instead of hitting the database to requery on id.  I guess the side effect of that was that it keeps all of it's memory around, which wasn't happening prior to that.  cc @kbrock